### PR TITLE
Update README.md - add ref to nf-core/rnaseq and Affymetrix

### PR DESCRIPTION
**Pull request by smoe**
I found your nf-core libraries via your cut'n'run library, which at the end did not tell me what to do with the scores. And the same then happened to me after I had executed the RNA-seq pipeline. In my mind, both those workflows should prepare for running this differential-abundance one afterwards. I had prepared https://github.com/nf-core/cutandrun/pull/198 with a rough description on how I had tried to make sense of it all (when I did not yet know about this workflow) and have prepared https://github.com/nf-core/rnaseq/pull/1060/ to at least provide a link to here.
 
I can very much understand if you do not want to make workflows inter-dependent at this stage, on a technical level, but some guidance of your users where to go next does not hurt.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
 ## Introduction
 
-**nf-core/differentialabundance** is a bioinformatics pipeline that can be used to analyse data represented as matrices, comparing groups of observations to generate differential statistics and downstream analyses. The initial feature set is built around RNA-seq, but we anticipate rapid expansion to include other platforms.
+**nf-core/differentialabundance** is an advanced bioinformatics pipeline that can be used to analyse data represented as matrices, comparing groups of observations to generate differential statistics and downstream analyses. The pipeline already supports RNA-seq data, as demonstrated in the nf-core [rnaseq workflow](https://github.com/nf-core/rnaseq), and Affymetrix/ThermoFisher .CEL files. However, our team is actively working on expanding its capabilities to encompass a broader range of platforms and maintain an effective tool to explore gene expression with ease and accuracy.
```
Comment (pinin4fjords) [inline README.md:15]:
```suggestion
**nf-core/differentialabundance** is a bioinformatics pipeline that can be used to analyse data represented as matrices, comparing groups of observations to generate differential statistics and downstream analyses. The pipeline supports RNA-seq data such as that generated by the nf-core [rnaseq workflow](https://github.com/nf-core/rnaseq), and Affymetrix arrays via .CEL files. Other types of matrix may also work with appropriate changes to parameters, and PRs to support additional specific modalities are welcomed. 
```

**Comment by pinin4fjords** [comment]
@nf-core-bot fix linting

**Comment by pinin4fjords** [comment]
@smoe thanks for the contribution- I would prefer the updated wording I committed. Could you add a changelog entry under the current 'added' please, following the template you will see used by other entries?

**Comment by smoe** [comment]
Done. Should I also rebase and squash?

File: CHANGELOG.md [hunk 2]
```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - [[#105](https://github.com/nf-core/differentialabundance/pull/105)] - Enabled multiple GMT/GMX files for GSEA ([@WackerO](https://github.com/WackerO), reported by [@grst](https://github.com/grst), review by [@pinin4fjords](https://github.com/pinin4fjords))
 - [[#108](https://github.com/nf-core/differentialabundance/issues/108)] - Add shiny app generation (starting feature set) ([@pinin4fjords](https://github.com/pinin4fjords), review by [@WackerO](https://github.com/WackerO))
 - [[#110](https://github.com/nf-core/differentialabundance/pull/110)] - Add shiny app outputs to tower.yml ([@pinin4fjords](https://github.com/pinin4fjords), review by [@WackerO](https://github.com/WackerO), [@maxulysse](https://github.com/maxulysse))
+- [[#149]](https://github.com/nf-core/differentialabundance/pull/149) - Update README.md - add ref to nf-core/rnaseq and Affymetrix ([@smoe](https://github.com/smoe), review by [@pinin4fjordas](https://github.com/pinin4fjords))
```
Comment (pinin4fjords) [inline CHANGELOG.md:14]:
```suggestion
- [[#149]](https://github.com/nf-core/differentialabundance/pull/149) - Update README.md - add ref to nf-core/rnaseq and Affymetrix ([@smoe](https://github.com/smoe), review by [@pinin4fjords](https://github.com/pinin4fjords))
```

**Review by pinin4fjords — APPROVED** [review]
Looks good now- just merge when the tests pass (no squash/ rebase required).

**Comment by smoe** [comment]
I do not have the permissions to merge, please kindly help me out - and also thank you for fixing the wording. I am confident that this will help many people.

The next thing for me, so I thought, would now be to have a look at the documentation and describe in more detail how to transition from the nf-core/rnaseq workflow.

**Pull request merged**